examples/helloworld/greeter_server/main.go

```diff
@@ -18,6 +18,7 @@
 
 //go:generate protoc -I ../helloworld --go_out=plugins=grpc:../helloworld ../helloworld/helloworld.proto
 
+// greeter_server command ...
```

Comment:
As this is a package level comment, it should start with "Package main ..."
```suggestion
// Package main implements a server for Greeter service. 
```